curvelops/curvelops.py

```diff
@@ -2,7 +2,7 @@
 Provides a LinearOperator for the 2D and 3D curvelet transforms.
 """
 
-__version__ = "0.2"
+__version__ = "0.21"
```

Comment:
Maybe at some point it would be good to switch to `setuptools_scm` or is there any problem with it?

Reply:
I will look into it! I have never done it, so would appreciate any guidance.

Reply:
Its fairly simple. You need `setuptools_scm` in your list of dependencies and then in the setup file you basically add this:

```
use_scm_version=dict(
        root=".", relative_to=__file__, write_to=src("curvelops/version.py")
    ),
setup_requires=["pytest-runner", "setuptools_scm"],
``` 

the file `curvelops/version.py` should not be there, it is made at build/install time for you and the version is consistent with your tags. I am not (yet) familiar with `pyproject.toml` so probably something can be done there as well instead of the setup.py file, maybe...

Also look at the bottom of this file https://github.com/PyLops/pylops/blob/dev/pylops/__init__.py, there is some boilerplate code to get the version correct when you do `curvelops.__version__`